pkg/cassandra/gocql/gocql.go

```diff
@@ -38,7 +38,11 @@ func WrapCQLSession(session *gocql.Session) CQLSession {
 
 // Query delegates to gocql.Session#Query and wraps the result as Query.
 func (s CQLSession) Query(stmt string, values ...interface{}) cassandra.Query {
-	return WrapCQLQuery(s.session.Query(stmt, values...))
+	q := s.session.Query(stmt, values...)
+	// https://github.com/gocql/gocql/issues/522
+	// we use IN and ORDER BY in multiple queries.
+	q.PageSize(0)
```

Comment:
I understand, but it is breaking the abstraction. The `cassandra.Session` is a generic wrapper around gocql to make it testable, it's not a place to apply defaults like this. In the actual reader we're going through a single method anyway, so we can add PageSize call there.